training/culture-and-process/CULTURE.md

```diff
@@ -0,0 +1,48 @@
+Culture in DP
+===========================
+Our culture and how we work is important for us to create an inclusive, empathetic and successful environment that works for everybody. 
+
+
+### Principles
+
+#### Inclusion
+We believe people from different backgrounds, with different identities and experiences, make our product, and our team, better. 
+
+We know a happy, comfortable and safe environment where our people are free to express themselves and their ideas means we create the best experiences for our users. 
+
+#### User first
+We put out users at the heart of everything we do. This includes external and internal users. We know our users come from everywhere with a range of expertise and needs which helps to create a better product for all.
+
+We follow [GDS design principles](https://www.gov.uk/guidance/government-design-principles#start-with-user-needs). 
+
+#### Open communication and collaboration
+We code in the open. This means most of our code is publicly available on Github and it hosts a lot of our coding discussions. When reviewing code we should we keep our comments constructive, helpful and kind.
+
+We encourage as many conversations and calls as possible to happen in open slack channels so that they are inclusive to all and can act as a source of information in the future. To let others know your concentrating on a task or that you'd like to minimise distractions use the headphones emoji 🎧 .
```

Comment:
Typo: Should be 'you are" or "you're" instead of "To let others know your concentrating…"

Reply:
👍